applications/ShapeOptimizationApplication/test_examples/01_Strain_Energy_Minimization_3D_Hook/materials_3D.json

```diff
@@ -1,14 +1,14 @@
 {
     "properties": [{
-            "model_part_name": "Parts_solid",
+            "model_part_name": "3D_Hook.Parts_solid",
             "properties_id": 0,
             "Material": {
                     "name": "Material",
                     "constitutive_law": {
                             "name": "KratosMultiphysics.StructuralMechanicsApplication.LinearElastic3DLaw"
                     },
                     "Variables": {
-                            "KratosMultiphysics.DENSITY": 7.85000E+03,                            
+                            "KratosMultiphysics.DENSITY": 7.85000E+03,
```

Comment:
you could remove all the `KratosMultiphysics.` and `KratosMultiphysics.StructuralMechanicsApplication.`, they are ignored anyway